external/tomcat/test.sh

```diff
@@ -16,6 +16,30 @@ source $(dirname "$0")/test_base_functions.sh
 #Set up Java to be used by the tomcat test
 echo_setup
 
+export CURRENT=`pwd`
+git clone -q -b 1.6.x --single-branch https://github.com/apache/apr.git $CURRENT/tmp/apr
+cd $CURRENT/tmp/apr
+./buildconf
+./configure --prefix=$CURRENT/tmp/apr-build
+make
+make install
+export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$CURRENT/tmp/apr-build/lib"
+
+git clone -q https://github.com/apache/tomcat-native.git $CURRENT/tmp/tomcat-native
+cd $CURRENT/tmp/tomcat-native/native
+sh buildconf --with-apr=$CURRENT/tmp/apr
+./configure --with-apr=$CURRENT/tmp/apr --with-java-home=$JAVA_HOME --with-ssl=yes --prefix=$CURRENT/tmp/tomcat-native-build
+make
+make install
+cd $CURRENT
+yes | cp build.properties.default build.properties
+echo >> build.properties
+echo "test.threads=4" >> build.properties
+echo "test.relaxTiming=true" >> build.properties
+echo "test.excludePerformance=true" >> build.properties
+echo "test.openssl.path=/dev/null/openssl" >> build.properties
+echo "test.apr.loc=$CURRENT/tmp/tomcat-native-build/lib" >> build.properties
+
 set -e
 echo "Building tomcat" && \
 cp build.properties.default build.properties && \
```

Comment:
The line should be removed as we have copied and updated build.properties file. This line might override what we have done.

Reply:
Line 45 removed, thank you!